### PR TITLE
Add WarioWare Gold

### PR DESCRIPTION
**Pull request by Hugman76**
Adds WarioWare Gold.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/warioware-gold/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "WarioWare Gold"
+description = "Similarly to past entries of the series, in WarioWare Gold, the player must complete consecutive \"microgames\" at increasing paces. Gold features microgames from both past entries in the series, as well as new microgames, making a sum of 300 microgames, the most the series has had to date. In addition, the game is the first of the series to feature full voice acting."
```
Comment (Leo626) [inline games/warioware-gold/game.dat]: You should start the description with this sentence like how wikipedia has it.

> WarioWare Gold is an action video game developed and published by Nintendo, with assistance by Intelligent Systems, and is the eighth game in the WarioWare series.

File: games/warioware-gold/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+needs_system_files = true
+needs_shared_font = true
+
+[[releases]]
+title = "00040000001D1C00"
+region = "eur"
+release_date = "2018-08-03"
```
Comment (Leo626) [inline games/warioware-gold/game.dat]: Unless you are from the future, this is for the USA release date. Should be `2018-07-27`.

Comment (Hugman76) [reply games/warioware-gold/game.dat]: I found the game physically today in my local french store, so yeah should be what you just said.

**Comment by Leo626** [comment]
@Hugman76 Still need 1 more screenshot.

**Comment by robzombie91** [comment]
[releases]
titleid = "00040000001D1C00"
region = "all"
release_date = "2018-08-03"

**Comment by robzombie91** [comment]
also minigame.PNG should be renamed to minigame.png to correct travis issue

**Comment by Leo626** [comment]
@robzombie91 But they don't share the same release dates? USA, Europe and Japan all have different release dates. Just to be sure, it's best to wait till the the other regions are released.

**Comment by CodingKoopa** [comment]
Superseded by #413.

**Pull request closed without merging**